### ISSUE TITLE
adding a word to a custom dictionary takes too long

### ISSUE THREAD

**Issue by kostyfisik** [languagetool-org/languagetool-browser-addon#74]
Instead of simply  adding a word to the local dictionary and excluding similar errors from the list (which should like to look as an instant update) it looks like the plugin reloads data to LT site and back (so it takes few seconds before you can continue on checking).

**Comment by danielnaber** [comment]
Yes, that's the way it is currently implemented. Patches welcome.

**Comment by kostyfisik** [comment]
Sorry, contributing to ru grammar rules has already overloaded my time table....
BTW, adding a correction from a provided list seems to lead to the same behaviour, which looks to be a bit overhead (the text cannot be modified without closing the plugin popup, this way is can only be changed with a plugin.  The reload should be run just before to report "no error" (after the corrections it is possible that some other rules can trigger).
